test/integration/projects/controls/gcloud_cloudbuild.rb

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cloudbuild_project_id = attribute('cloudbuild_project_id')
+branches_regex = '^(development|non\\-production|production)$'
+cloud_source_repos = [
+  'app_infra_repo'
+]
+
+control 'gcloud_cloudbuild' do
```

Comment:
We should add this control here https://github.com/terraform-google-modules/terraform-example-foundation/blob/a3a413d623d7e5a312dcfa84a67e3ff528e14abf/.kitchen.yml#L114